Frontend/src/tests/home.test.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
Please call the file name the same name as the file it's testing. So in this case Home with an upper case. And usually it's called spec and not test so:
`Home.spec.tsx`